cmd/buildah/run.go

```diff
@@ -79,10 +79,11 @@ func runCmd(c *cli.Context) error {
 		Args:     c.StringSlice("runtime-flag"),
 	}
 
-	if c.Bool("tty") {
+	if c.IsSet("tty") {
 		options.Terminal = buildah.WithTerminal
-	} else {
-		options.Terminal = buildah.WithoutTerminal
+		if !c.Bool("tty") {
+			options.Terminal = buildah.WithoutTerminal
+		}
```

Comment:
@nalind I tested this code by adding a few println's (see below) and the second if statement here is invoked with tty==false and not in other cases.

```
        if c.IsSet("tty") {
                fmt.Println("TOM: In tty IsSet")
                options.Terminal = buildah.WithTerminal
                if !c.Bool("tty") {
                        fmt.Println("TOM: In tty not Boolean")
                        options.Terminal = buildah.WithoutTerminal
                }
        }

and saw:
[root@localhost buildah]# ./buildah run fedora-working-container /bin/bash --tty=false
TOM: In tty IsSet
TOM: In tty not Boolean
exit
[root@localhost buildah]# ./buildah run fedora-working-container /bin/bash --tty
TOM: In tty IsSet
[root@mrsdalloway /]# exit
exit
[root@localhost buildah]# ./buildah run fedora-working-container /bin/bash --tty=true
TOM: In tty IsSet
[root@mrsdalloway /]# exit
exit
[root@localhost buildah]# ./buildah run fedora-working-container /bin/bash 
[root@mrsdalloway /]# exit
exit
```